packages/common/src/services/submissionService.ts

```diff
@@ -142,6 +142,12 @@ const service = (dependencies: Dependencies) => {
 				inProcessEntities: entitiesToProcess,
 			};
 		},
+
+		activeSubmission: async (categoryId: number) => {
```

Comment:
should probably be called `getActiveSubmission` or similar.

Reply:
renamed to `getActiveSubmission`